deepchem/utils/voxel_utils.py

```diff
@@ -159,3 +161,52 @@ def voxelize(get_voxels: Callable[..., Any],
           feature_tensor[voxel[0], voxel[1], voxel[2], 0] += 1.0
 
   return feature_tensor
+
+
+def voxelize_pi_stack(prot_xyz, prot_rdk, lig_xyz, lig_rdk, distances,
```

Comment:
We should add type annotations here in a follow-up PR, but no need to on this PR since we want to get it in before the end of the week :)